students/BrianB/lesson04/mailroom2.py

```diff
@@ -1,3 +1,153 @@
-#! bin/usr/env python3
+#!/usr/bin/env python3
+
+# ----------------------------------------------
+# What: Lesson04: mailroom2.py
+# Who: Brian Brumleve
+# Date: May 10, 2019
+# Change Description:
+#       Updated with Mailroom#2 requirements,
+#       dictionaries are added.
+#
+# ----------------------------------------------
+
+import sys
+import math
+from textwrap import dedent
+
+# print("In the Mailroom")
+
+
+def gen_donor():
+    return [donor for donor in donor_data.keys()]
```

Comment:
You actually don't need to have the .keys(). When iterating through a dict, it will always do it by keys